### PR TITLE
Create Header Component

### PR DESCRIPTION
**Pull request by mAbadsa**
Relates #58
**Header Component.**
Create Header Component with links and titles.
1. Using Material-UI pre-built Component [ Container, Toolbar, Typography].
2. Using LinkItem #60  Component.
3. With CSS Styling.
4. We will use the Header component on every page.

### CODE REVIEW

**Review by yasmeenattallah — COMMENTED** [review]
why don't use prop-types

**Review by yasmeenattallah — COMMENTED** [review]
it's better to spread make a style in a style.js file

**Review by AdhamHaisami — DISMISSED** [review]

File: client/src/Components/Navbar/style.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
```
Comment (mariamisa) [inline client/src/Components/Navbar/style.js:3]: pass the theme as a parameter to your makeStyle function, then get color from the theme, palette
``` 
const useStyles = makeStyles((theme) => ({
```

**Review by mariamisa — CHANGES_REQUESTED** [review]
pull main

File: client/src/Components/Navbar/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  return (
+    <>
+      <Container maxWidth="lg">
+        <AppBar position="static" className={classes.root}>
+          <Toolbar className={classes.toolBar}>
+            <Typography variant="h6" className={classes.logo}>
+              House Hunting
+            </Typography>
+            <Typography className={classes.linkItem}>
+              <LinkItem underline="none" text="Houses" linkUrl="/" />
```
Comment (mariamisa) [inline client/src/Components/Navbar/index.js:20]: get link Url value from ``` client/src/Utils/routes.constant.js file ```

**Review by mariamisa — CHANGES_REQUESTED** [review]

**Review by AdhamHaisami — APPROVED** [review]

**Review by AdhamHaisami — DISMISSED** [review]

**Review by mariamisa — APPROVED** [review]

**Pull request merged**